README.md

```diff
@@ -1,45 +1,203 @@
+-5,Open the downloaded file<br />
+-6,Choose Installation directory<br />
+-7,Select Components<br />
+-8,Select Data Directory<br />
+-9,Set Password<br />
+-10,Set Port Number<br />
+-11,Advanced Options<br />
+-12,Verify Pre-installation Summary<br />
+-13,Begin PostgreSQL Installation<br />
+
+
+### Usage
+
+To run the project, open postgres and run the commands inside the data.sql,schema.sql,queries.sql folders:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Yilikal Teshome"></a>
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/Yilikal250)
+- Twitter: [@twitterhandle](https://twitter.com/TeshomeYilikal)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/yilikal-teshome)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
 
-## 🤝 Contributing
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project please give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse for giving us this platform and for organizing the files to learn. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[What is MySQL]**
+
+  - [MySQL is a database management system.]
+
+- **[What is the primary key?]**
+
+  - [A primary key, also called a primary keyword, is a column in a relational database table that's distinctive for each record.]
 
-Give a ⭐️ if you like this project!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Acknowledgments
+<!-- LICENSE -->
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
 
-This project is [MIT](./MIT.md) licensed.
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove guide note below it.